### DISCUSSION TITLE
Github API: (403), API rate limit exceeded for $my internet ip address$

### DISCUSSION THREAD

**Discussion by iamtheonlydave** [netalertx/NetAlertX#263] [Q&A]
I get the following message at the top of the Maintenance Tools

Github API: (403), API rate limit exceeded for 84.66.198.185. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)
Database-Path	/home/pi/pialert/db/pialert.db
Database-Size	0,64 MB
Last Modification	June 23 2023 16:16:36
DB Backups	0 backups were found / total disk usage: 0,00 MB

I've googled Github API 403 rate limit errors and haven't found any relevant articles. I'm thinking it is something simple and obvious but as I'm new to Github, here I am. Any help welcomed

**Comment by jokob-sk** [comment]
Hi there!

This should not be causing any functional issues as it only relates to the new version check. The rate limits are reset every couple of hours so the issue will eventually go away. 

j

**Reply by iamtheonlydave** [reply]
Ok, that makes sense. I was worried that I was breaking Github etiquette.
